### DISCUSSION TITLE
How can I use a wildcard certificate and a regular SSL certificate at the same time?

### DISCUSSION THREAD

**Discussion by vsfomin** [nginx-proxy/nginx-proxy#2526] [Q&A] [answered]
I have a bar.com domain and foo.bar.com domains
According to the documentation, I have to mount the wildcard certificate in the /etc/nginx/certs folder and name them bar.com.crt and bar.com.key
The standard certificate for the domain bar.com I should also mount there under the same names bar.com.crt and bar.com.key.
How can I use two types of certificate at the same time?

**Comment by buchdag** [comment]
If I understand correctly, you want to be able to use the single domain certificate for `bar.com` and the wildcard certificate for every other subdomains of `bar.com` like `foo.bar.com` ?

**Reply by vsfomin** [reply]
Yes, correct. File names are the same in this case.

**Reply by buchdag** [reply, marked as answer]
You can use the [`CERT_NAME`](https://github.com/nginx-proxy/nginx-proxy/tree/main/docs#sni) environment variable to force nginx-proxy to use a specific certificate for a given container or container set, rather than automatically selecting the certificate to use based on the domain name.

You can name you single domain certificate and key something like `_base.bar.com.crt` and `_base.bar.com.key`, mount them to `/etc/nginx/certs`, then use `CERT_NAME=_base.bar.com` on the container that should use it. You can then mount your wildcard certificate / key as `bar.com.crt` / `bar.com.key` and every subdomains of `bar.com` will automatically use it.

The other way is also possible, like in the docs example.
